### ISSUE TITLE
Dynamixel Pro code

### ISSUE THREAD

**Issue by jonghwan7028** [arebgun/dynamixel_motor#17]
Can you guys upgrade the current code for dynamixel pro?
I noticed that the pro has a different protocol that causes wrong communication. Thanks.

**Comment by rohit-murthy** [comment]
Hi, have you by any chance been able to get the code for a dynamixel pro?
Thanks.

**Comment by arebgun** [comment]
No, Dynamixel PRO is not supported. I don't have access to the new motors and I am not actively developing or adding new features.
